### PR TITLE
Ibm object storage

### PR DESCRIPTION
**Pull request by roytman**
Add a configuration option to use IBM Cloud Object Storage as Ark's backups destination.

### CODE REVIEW

**Comment by ncdc** [comment]
Hi @roytman, thanks for your PR! We'll review this soon and get back to you with any feedback we may have.

**Comment by ncdc** [comment]
@roytman we haven't forgotten about this! @Bradamant3 is busy at the moment with other things, but she'll hopefully be able to review starting this Wednesday. Thanks for your patience!

**Comment by roytman** [comment]
Hi Andy, thanks for the update. No problem, it can wait.

**Comment by roytman** [comment]
Hi Andy @ncdc, I updated the ibm-config.md according to the latest Ark v0.7.0 namespace policy.
P.S in the README.md the command 
`kubectl get deployments -l component=ark --namespace=heptio-ark` 
should be replaced by 
`kubectl get deployments -l component=ark --namespace=heptio-ark-server`

**Comment by Bradamant3** [comment]
My apologies! I thought that I had reviewed here, so sorry.
Docs LGTM, with additional apologies for merging a change that means you need to rebase :-(
I'd like to make a few copyedits, but let's get this PR merged first, and then I can ask @roytman to review my changes in a subsequent PR. My changes will be stylistic only, but it's always good to make sure I don't accidentally change technical meaning.

**Comment by ncdc** [comment]
@roytman thanks for the info about the readme. We'll get that fixed. If you can rebase this PR, we'll merge it asap.

**Comment by ncdc** [comment]
@roytman it looks like you're having git issues? Need any help?

**Comment by roytman** [comment]
@ncdc thanks for the help suggestion. The PR was rebased.

File: examples/ibm/00-ark-config.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  bucket: <YOUR_BUCKET>
+  config:
+    region: <YOUR_REGION>
+    s3ForcePathStyle: "true"
+    s3Url:  <YOUR_URL_ACCESS_POINT>
+backupSyncPeriod: 1m
+gcSyncPeriod: 1m
+scheduleSyncPeriod: 1m
+restoreOnlyMode: false
+---
```
Comment (ncdc) [inline examples/ibm/00-ark-config.yaml]: This line isn't required

File: examples/ibm/00-ark-config.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  namespace: heptio-ark
+  name: default
+backupStorageProvider:
+  name: aws
+  bucket: <YOUR_BUCKET>
+  config:
+    region: <YOUR_REGION>
+    s3ForcePathStyle: "true"
+    s3Url:  <YOUR_URL_ACCESS_POINT>
+backupSyncPeriod: 1m
```
Comment (ncdc) [inline examples/ibm/00-ark-config.yaml]: The other examples have `30m` - want to be consistent?

File: examples/ibm/00-ark-config.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+gcSyncPeriod: 1m
```
Comment (ncdc) [inline examples/ibm/00-ark-config.yaml]: The other examples have `30m` - want to be consistent?

File: examples/ibm/00-ark-config.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2017 the Heptio Ark contributors.
```
Comment (ncdc) [inline examples/ibm/00-ark-config.yaml]: 2018 😄

**Comment by roytman** [comment]
Hi @ncdc, thanks for the comments. I updated the file.

**Comment by ncdc** [comment]
LGTM. Please squash & I'll merge. Thanks!

**Comment by roytman** [comment]
Done

**Pull request merged**